play/plugin/src/test/kotlin/com/github/triplet/gradle/play/tasks/shared/PublishArtifactIntegrationTests.kt

```diff
@@ -74,7 +74,7 @@ interface PublishArtifactIntegrationTests : SharedIntegrationTest {
         assertArtifactUpload(result1)
 
         result2.requireTask(outcome = SUCCESS)
-        assertThat(result2.output).contains("didPreviousBuildSkipCommit=true")
+        assertThat(result2.output).contains("didPreviousBuildSkipCommit=false")
```

Comment:
I'm not fully getting what this test suposted to do. It started to fail after introducing the changes on `PlayApiService` (probably the `AutoCloseable` + `usesService`).

Based on how it looks, should `play.commit` in line 66 be `true` already?  

Anyway, `didPreviousBuildSkipCommit` seems now to be always false, which as far I understand, makes sense, since there no possibility the that the previous build skiped the commit, since it's going to be cancelled anyway at the end when `AutoCloseeable.close` is called.

Should `commit = false` imply `cleanupAtClose = false` aswell?

Reply:
> Should commit = false imply cleanupAtClose = false as well?
I'm going to assume that for now, so the test will pass. I think it makes sense.

Reply:
Yes, commit false allows you to keep a pending publishing action across builds.